docs/CONFIG-PROPERTIES.md

```diff
@@ -70,11 +66,37 @@
+
+Due to implementation specifics, there are two different sets of log levels that can be set: logrus and syslog levels.
+Logrus levels are used by the EVE microservices, while syslog levels are used by syslog and kernel.
+
+* the logrus levels are as follows: panic, fatal, error, warning, info, debug, and trace ["https://pkg.go.dev/github.com/sirupsen/logrus"].
+* the syslog levels are as follows: emerg, alert, crit, err, warning, notice, info, debug ["https://man7.org/linux/man-pages/man3/syslog.3.html"].
+
+Additionally all log levels can be set to "none" to disable logging for the corresponding component or to "all" to enable all log levels.
+
+Furthermore, the "remote" log levels control, which subset of the produced logs will be sent to the controller.
+A corresponding "remote" log level can be set for each of the three components: EVE microservices, syslog, and kernel.
+
+| Name | Type | Default | Description |
+| ---- | ---- | ------- | ----------- |
+| debug.default.loglevel | string | debug | default level of logs produced by EVE microservices. Can be overwritten by agent.*agentname*.debug.loglevel. Uses logrus log levels as described here ["https://pkg.go.dev/github.com/sirupsen/logrus"]: panic, fatal, error, warning, info, debug and trace.
+| debug.default.remote.loglevel | string | warning | default level of logs sent by EVE microservices to the controller. Can be overwritten by agent.*agentname*.debug.remote.loglevel. Uses logrus log levels as described here ["https://pkg.go.dev/github.com/sirupsen/logrus"]: panic, fatal, error, warning, info, debug and trace. |
+| debug.syslog.loglevel | string | info | level of the produced syslog messages. System default loglevel string representation should be used as described here ["https://man7.org/linux/man-pages/man3/syslog.3.html"]: emerg, alert, crit, err, warning, notice, info, debug. |
+| debug.syslog.remote.loglevel | string | info | level of the syslog messages sent to the controller. System default loglevel string representation should be used as described here ["https://man7.org/linux/man-pages/man3/syslog.3.html"]: emerg, alert, crit, err, warning, notice, info, debug. |
+| debug.kernel.loglevel | string | info | level of the produced kernel log messages. System default loglevel string representation should be used as described here ["https://man7.org/linux/man-pages/man3/syslog.3.html"]: emerg, alert, crit, err, warning, notice, info, debug. |
+| debug.kernel.remote.loglevel | string | info | level of the kernel log messages sent to the controller. System default loglevel string representation should be used as described here ["https://man7.org/linux/man-pages/man3/syslog.3.html"]: emerg, alert, crit, err, warning, notice, info, debug. |
+
+In addition, there can be per-agent settings to overwrite the default log level set for zedbox.
```

Comment:
s/zedbox/EVE microservices/ (even though this only applies to some microservices like downloader and zedagent and not to others like wwan and wlan.

Reply:
sorry, I meant to replace all of the occurrences and forgot that one

Reply:
actually this applies to wwan as well
https://github.com/lf-edge/eve/blob/08d0583f68c9ce35747d2ea6566a9b30ad7404c6/pkg/wwan/mmagent/agent.go#L486

Reply:
here is the full list of all "agents" that support log level settings (call `agentlog.HandleGlobalConfig`)  right now:
newlogd
wwan
nodeagent
downloader
tpmmgr
client
vcomlink
executor
vaultmgr
baseosmgr
zedagent
verifier
wstunnelclient
zfsmanager
zedkube
ledmanager
faultinjection
zedmanager
nim
loguploader
watcher
volumemgr
zedrouter
msrv
domainmgr
diag

Reply:
I added it also to `CONFIG-PROPERTIES.md`